src/nvim/CMakeLists.txt

```diff
@@ -552,21 +552,24 @@ if(CLANG_ASAN_UBSAN)
   else()
     set(SANITIZE_RECOVER -fno-sanitize-recover)         # Clang 3.5-
   endif()
-  set_property(TARGET nvim APPEND_STRING PROPERTY COMPILE_FLAGS "-DEXITFREE ")
-  set_property(TARGET nvim APPEND_STRING PROPERTY COMPILE_FLAGS "${SANITIZE_RECOVER} -fno-omit-frame-pointer -fno-optimize-sibling-calls -fsanitize=address -fsanitize=undefined -fsanitize-blacklist=${PROJECT_SOURCE_DIR}/src/.asan-blacklist")
+  set_property(TARGET nvim APPEND PROPERTY COMPILE_DEFINITIONS "EXITFREE")
```

Comment:
👍  since we bumped minimum CMake to 2.8.12 https://github.com/neovim/neovim/pull/4389 no longer applies.